examples/CreatePerTitleEncodingDashAndHls.php

```diff
@@ -0,0 +1,398 @@
+<?php
+
+use Bitmovin\api\ApiClient;
+use Bitmovin\api\enum\AclPermission;
+use Bitmovin\api\enum\CloudRegion;
+use Bitmovin\api\enum\codecConfigurations\H264Profile;
+use Bitmovin\api\enum\SelectionMode;
+use Bitmovin\api\enum\Status;
+use Bitmovin\api\enum\manifests\dash\DashMuxingType;
+use Bitmovin\api\enum\manifests\hls\MediaInfoType;
+use Bitmovin\api\exceptions\BitmovinException;
+use Bitmovin\api\model\codecConfigurations\AACAudioCodecConfiguration;
+use Bitmovin\api\model\codecConfigurations\H264VideoCodecConfiguration;
+use Bitmovin\api\model\encodings\AutoRepresentation;
+use Bitmovin\api\model\encodings\Encoding;
+use Bitmovin\api\model\encodings\EncodingMode;
+use Bitmovin\api\model\encodings\helper\Acl;
+use Bitmovin\api\model\encodings\helper\EncodingOutput;
+use Bitmovin\api\model\encodings\helper\InputStream;
+use Bitmovin\api\model\encodings\muxing\FMP4Muxing;
+use Bitmovin\api\model\encodings\muxing\TSMuxing;
+use Bitmovin\api\model\encodings\muxing\MuxingStream;
+use Bitmovin\api\model\encodings\pertitle\H264PerTitleConfiguration;
+use Bitmovin\api\model\encodings\pertitle\PerTitle;
+use Bitmovin\api\model\encodings\StartEncodingRequest;
+use Bitmovin\api\model\encodings\streams\Stream;
+use Bitmovin\api\model\encodings\streams\StreamMode;
+use Bitmovin\api\model\inputs\S3Input;
+use Bitmovin\api\model\outputs\S3Output;
+use Bitmovin\api\model\manifests\dash\AudioAdaptationSet;
+use Bitmovin\api\model\manifests\dash\DashManifest;
+use Bitmovin\api\model\manifests\dash\DashRepresentation;
+use Bitmovin\api\model\manifests\dash\Period;
+use Bitmovin\api\model\manifests\dash\VideoAdaptationSet;
+use Bitmovin\api\model\manifests\hls\HlsManifest;
+use Bitmovin\api\model\manifests\hls\MediaInfo;
+use Bitmovin\api\model\manifests\hls\StreamInfo;
+
+use Ramsey\Uuid\Uuid;
```

Comment:
This is package is only stated as a develop requirement in composer.json and therefore doesn't get installed by default as `composer install --dev` is needed to do so.

Therefore it would break the execution of the example, when the API client gets installed as instructed in README